src/wizard/wizard-form.tsx

```diff
@@ -43,6 +44,14 @@ export default function WizardForm({
   const isLastStep = activeStepIndex >= steps.length - 1;
   const skipToTargetIndex = findSkipToTargetIndex(steps, activeStepIndex);
   const isMobile = useMobile();
+  const stepHeaderRef = useRef<HTMLDivElement | null>(null);
+
+  useEffectOnUpdate(() => {
+    if (stepHeaderRef && stepHeaderRef.current) {
+      const header = stepHeaderRef.current.querySelector('[tabindex = "-1"]') as HTMLElement;
+      header?.focus();
+    }
+  }, [activeStepIndex]);
```

Comment:
With that the header will also be focused when the navigation happens programmatically. That might not be safe.

Imagine the page loads and the active step index is determined after some delay - the navigation in that case might or might not be expected. We might want to let customers control that by offering a function like `wizardRef.current.focusStepHeader()`.